Makefile

```diff
@@ -29,13 +31,14 @@ release:
 	GOOS=windows GOARCH=386 go build ${SET_VERSION} -o ./bin/${BINARY}_${VERSION}_windows_386
 	GOOS=windows GOARCH=amd64 go build ${SET_VERSION} -o ./bin/${BINARY}_${VERSION}_windows_amd64
 
-install: build
-	mkdir -p ${PLUGIN_DIR}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
-	cp ${BINARY} ${PLUGIN_DIR}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+install: compile
```

Comment:
Even though this works fine when executed manually, for some reason, when executing this on the Buildkite agent, the dependent target was not being triggered when it was named 'build'. Not sure if that's some sort of reserved word or ??

Reply:
Build log demonstrating the behavior: https://buildkite.com/scaleft/terraform-provider-oktapam-artifact/builds/58#01828e93-bdd8-4ba2-96ec-06db95c9d1d0

Reply:
Probably just needs a marker to indicate it's a phony target.